frontend/src/components/VBackToSearchResultsLink.vue

```diff
@@ -26,9 +30,37 @@ export default defineComponent({
     VIcon,
     VLink,
   },
+  props: {
```

Comment:
These must be added below the `inheritAttrs` field to match the [recommended option order](https://eslint.vuejs.org/rules/order-in-components.html).

Also, if you add new required props to a component, you will need to update the usage of that component. This particular component `VBackToSearchResultsLink.vue` is used in the following locations, which need to be updated:

- `frontend/src/pages/audio/_id/index.vue`
- `frontend/src/pages/image/_id/index.vue`
- `frontend/src/components/meta/VBackToSearchResultsLink.stories.mdx`